src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_SIZE = 10;
+    private Object[] array;
+    private int size;
+    
+    public ArrayList() {
+        this.array = new Object[INITIAL_SIZE];
+    }
+    
     @Override
     public void add(T value) {
-
+        if (size == array.length - 1) {
+            increaseArray(array.length + array.length / 2 + 1);
```

Comment:
```suggestion
            increaseArray();
```
You use this method two times and pass the same argument. So, why do you need this argument.
You can calculate array.length + array.length / 2 inside increaseArray()
And why do you need '+1' by the way?

Reply:
Hi! I need plus 1 beacause I think that I need grow up arraylist and add the added element.
Is it not correct? I found artical about this on internet. Maybe it was wrong.